src/app/page.tsx

```diff
@@ -2,6 +2,7 @@
 import { useEffect } from "react";
 import styles from "./page.module.css";
 import MainSection from "./section/mainSection/MainSection";
+import FeatureSection from "./section/FeatureSection/FeatureSection";
 
 export default function Home() {
```

Comment:
@Dheeru12503 
change this default export take reference from headerComponent 
TODO : ( create const function then export at last line )

move the `Gallery` function outside the Home component 
also rename the component name to DefaultPage (this will make more sense)

Reply:
done

Reply:
@Dheeru12503 
same here push your changes

Reply:
done